platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml

```diff
@@ -50,6 +50,7 @@ network:
     - organization:
       name: carrier
       type: creator       # creator organization will create the channel and instantiate chaincode, in addition to joining the channel and install chaincode
+      org_status: new
```

Comment:
is this needed in Channels section as well?

Reply:
yes, because of the tag we can know the newest member of the channel and also most of the roles were based on the channel loops to reuse them it is required